### PR TITLE
#425 Apply FINOS Software Project Blueprint

### PR DESCRIPTION
**Pull request by symphony-thibault**
Related issue: https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/issues/425

Steps required by https://odp.finos.org/docs/project-collaboration/#finos-project-blueprint: 
- [x] A README.md template, with a suggested structure of content
- [x] A NOTICE template, required by FINOS bylaws
- [x] A LICENSE template, required by FINOS bylaws
- [x] A Code of Conduct template, required by FINOS bylaws
- [x] Contributing guidelines, required by FINOS bylaws
- [x] WhiteSource custom configuration, to setup continuous security scanning for your project
- [x] Issue templates for bugs, feature requests and support questions , defined in the .github folder

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,35 +1,55 @@
-# How to contribute
+# Symphony BDK for Java Contribution and Governance Policies
 
-If you found a bug or issue, please ensure the bug was not already reported by searching in
-[GitHub issues](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/issues).
-If you are unable to find an open issue addressing the problem, open a new one. 
-Be sure to include a title and clear description, the SDK version, and a code sample demonstrating the issue.
+This document describes the contribution process and governance policies of the FINOS {project name} project. The project is also governed by the [Linux Foundation Antitrust Policy](https://www.linuxfoundation.org/antitrust-policy/), and the FINOS [IP Policy](IP-Policy.pdf), [Code of Conduct](Code-of-Conduct.md), [Collaborative Principles](Collaborative-Principles.md), and [Meeting Procedures](Meeting-Procedures.md).
```
Comment (symphony-elias) [inline CONTRIBUTING.md:3]: Please review the links and put absolute paths.

Comment (maoo) [reply CONTRIBUTING.md:3]: 👍 ty for pointing that out, I fixed our project-blueprint accordingly.

```
This document describes the contribution process and governance policies of the FINOS {project name} project. The project is also governed by the [Linux Foundation Antitrust Policy](https://www.linuxfoundation.org/antitrust-policy/), and the FINOS [IP Policy](https://github.com/finos/community/blob/master/governance/IP-Policy.pdf), [Code of Conduct](https://github.com/finos/community/blob/master/governance/Code-of-Conduct.md), [Collaborative Principles](https://github.com/finos/community/blob/master/governance/Collaborative-Principles.md), and [Meeting Procedures](https://github.com/finos/community/blob/master/governance/Meeting-Procedures.md).
```

Comment (maoo) [reply CONTRIBUTING.md:3]: Please replace all `{project name}` with `symphony-bdk` - assuming this is the name repository you want to give to the GitHub repository, after being transferred to github.com/finos

File: LICENSE [hunk 2]
```diff
@@ -1,21 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright {yyyy} {name of copyright owner}
```
Comment (maoo) [inline LICENSE:189]: please replace all `{yyyy}` with `2021` and `{name of copyright owner}` with `Symphony LLC`

Make sure that you replaced all `{}` placeholders.

**Review by maoo — CHANGES_REQUESTED** [review]
Great work! Few small notes:
1. Found some places where the `{}` placeholders are not replaced
2. I can't find the NOTICE file
3. `whitesource.config` is not needed, as you're using Snyk now

All the rest seems to be in place.

Thank you!

**Review by symphony-youri — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (symphony-thibault) [reply CONTRIBUTING.md:3]: `symphony-bdk-java` is OK? Assuming that `symphony-bdk-python` will follow later :)

**Pull request merged**